Cargo.toml

```diff
@@ -1,23 +1,23 @@
 [workspace]
 
 members = [
-    "crates/acvm_backend_barretenberg",
-    "crates/noirc_evaluator",
-    "crates/noirc_frontend",
-    "crates/noirc_errors",
-    "crates/noirc_driver",
-    "crates/noirc_printable_type",
-    "crates/nargo",
-    "crates/nargo_cli",
-    "crates/nargo_toml",
-    "crates/fm",
-    "crates/arena",
-    "crates/noirc_abi",
-    "crates/noirc_abi_wasm",
-    "crates/iter-extended",
-    "crates/wasm",
+    "compiler/noirc_evaluator",
+    "compiler/noirc_frontend",
+    "compiler/noirc_errors",
+    "compiler/noirc_driver",
+    "compiler/noirc_printable_type",
+    "tooling/acvm_backend_barretenberg",
+    "tooling/nargo",
+    "tooling/nargo_cli",
+    "tooling/nargo_toml",
+    "compiler/fm",
+    "compiler/arena",
+    "compiler/noirc_abi",
+    "compiler/noirc_abi_wasm",
```

Comment:
These are tooling in my mind

Reply:
Currently noirc_driver and noirc_evaluator rely on the ABI, so I was hesitant to move that into tooling. If its meant to be in tooling, then we definitely have the wrong abstraction with respects to ABI handling

Reply:
Yeah, I believe the current abstraction is tangled currently. It should be easy to remove from noirc_driver as it is only used for the `compute_function_abi` function, but the `noirc_evaluator` needs a bunch of work.

Reply:
I'll move it into tooling since it won't cause cyclic dependencies and then open an issue to untangle the current code 👍